middleware/jwt.go

```diff
@@ -123,7 +129,7 @@ func JWTWithConfig(config JWTConfig) echo.MiddlewareFunc {
 	if config.Skipper == nil {
 		config.Skipper = DefaultJWTConfig.Skipper
 	}
-	if config.SigningKey == nil && len(config.SigningKeys) == 0 {
+	if config.SigningKey == nil && len(config.SigningKeys) == 0 && config.KeyFunc == nil {
```

Comment:
I guess you want also limit because a custom KeyFunc may ignore the signing keys altogether.
That should definitly be documented too, that a KeyFunc must take care of signature validation too.

Reply:
What do you mean with "limit"? any suggestion where it should be documented?

Yes, the KeyFunc must take care of the signature validation and indeed it is not in the control of the middleware.

Reply:
Forget about "limit" that was just a pretty bad translation of a non-native speaker ;-)

Docs reside in the [labstack/echox](https://github.com/labstack/echox) project, which is used to create https://echo.labstack.com/cookbook/jwt

So a PR there to add an example for `KeyFunc`and a warning to be sure to actually check the signature would be great. Could you do that PR too?

Reply:
I will create a PR in the other repository, most likely at the end of next week. Please bear with me (busy with work and family). Thank you!

Reply:
I noticed the JWT middleware documentation in the echox project is not up to date when comparing with the supported JWTConfig. I will make them synchronized too.